Program.cs

```diff
@@ -13,55 +12,40 @@
 
 var builder = WebApplication.CreateBuilder(args);
 
-try
-{
-    var path = Path.Combine(builder.Environment.ContentRootPath, ".env");
-    DotEnv.Load(path);
-}
-catch (Exception ex)
-{
-    Console.WriteLine($"Error loading .env file: {ex.Message}");
-}
+// load local dotnet user-secrets
+builder.Configuration.AddUserSecrets("d70ac473-6c10-438a-a3ba-2a154bdf5946");
```

Comment:
ahhh I see now

Reply:
I didn't have to do this line for the template at work, but it does what we need it to do